src/database/migrations/1731879052218-migration-name-placeholder.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddWishlistOnUserTable1731879052218 implements MigrationInterface {
+  name = 'AddWishlistOnUserTable1731879052218';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "wishlist" text`);
```

Comment:
Should it be text or another type ? Why didn't you think about many to many relationship

Reply:
I thought about many to many relationship, but we do not need to display user on accommmodation, it works properly for user toggle like accommodation,so i chose this way